src/repos.rs

```diff
@@ -96,8 +106,8 @@ impl RepoConfig {
         conn.execute(
             r#"INSERT INTO repos (repo, channel, force_push_notify, force_push_reapply_statuses,
                                   branches, jira_projects, jira_versions_enabled, version_script,
-                                  release_branch_prefix)
-               VALUES (?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9)"#,
+                                  release_branch_prefix, next_branch_suffix)
+               VALUES (?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9, ?10)"#,
```

Comment:
painful magic numbers :) see them listed below...

Reply:
yeah... didn't know how to do named params. i should look into that again.